contracts/wrapper/NameWrapper.sol

```diff
@@ -560,21 +573,25 @@ contract NameWrapper is
         return fuses & CANNOT_TRANSFER == 0;
     }
 
-    function _makeNode(bytes32 node, bytes32 label)
+    function _makeNode(bytes32 node, bytes32 labelhash)
         private
         pure
         returns (bytes32)
     {
-        return keccak256(abi.encodePacked(node, label));
+        return keccak256(abi.encodePacked(node, labelhash));
     }
 
     function _addLabel(string memory label, bytes memory name)
         internal
         pure
         returns (bytes memory ret)
     {
-        require(bytes(label).length > 0, "NameWrapper: Label too short");
-        require(bytes(label).length < 256, "NameWrapper: Label too long");
+        if (bytes(label).length < 1) {
+            revert LabelTooShort("");
```

Comment:
Since this will always be the empty string, we can just leave out the argument entirely.